IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeVoltmeter/DocPropertySet.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_FlowInstrumentTypeVoltmeter" UniqueId="4b47169f-04e8-f329-e589-ca2bfe243b09" ApplicableType="IfcFlowInstrument/VOLTMETER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="SecondaryVoltage_0jvGIYxLz90vG7yrY_qQ8T" />
```

Comment:
this is the wrong secondaryVoltage
```suggestion
		<DocProperty xsi:nil="true" href="SecondaryVoltage_1ZIvG0qX4HuO00025QrEV" />
```

Reply:
0ed68dc80cfee0e7a216fa8ce063c47b9d42fe5f